cyctl/.goreleaser.yaml

```diff
@@ -0,0 +1,82 @@
+version: 1
+
+env:
+  - CGO_ENABLED=0
+
+builds:
+  - id: linux
+    binary: cyctl
+    goos:
+      - linux
+      - freebsd
+    goarch:
+      - amd64
+      - arm64
+      - arm
+      - ppc64le
+      - s390x
+    goarm:
+      - 7
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X github.com/cyclops-ui/cyclops/common.CliVersion={{.Version}}
+
+  - id: osx
+    binary: cyctl
+    goos:
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X github.com/cyclops-ui/cyclops/common.CliVersion={{.Version}}
+
+  - id: windows
+    binary: cyctl
+    goos:
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    flags:
+      - -trimpath
+    ldflags:
+      - -s -w -X github.com/cyclops-ui/cyclops/common.CliVersion={{.Version}}
+
+archives:
+  - format: tar.gz
+    name_template: "cyctl_{{ .Os }}_{{ .Arch }}"
```

Comment:
@SpiffyEight77  Can we include the version to the name? Same goes for nfpm

Reply:
> @SpiffyEight77 Can we include the version to the name? Same goes for nfpm

@petar-cvit

Sure, we can add the version to the name like `cyctl_darwin_arm64_v0.6.2.tar.gz` or use any other naming format you prefer.

I'll handle this later tonight after work.

Reply:
Yeah, this looks good, thanks

Reply:
The version is included in the file name. For more details, please visit the [latest release](https://github.com/SpiffyEight77/cyclops/releases/tag/v0.6.11) in the forked repository.